app/controllers/concerns/foreman_monitoring/find_host_by_client_cert.rb

```diff
@@ -7,6 +7,7 @@ module FindHostByClientCert
     module ClassMethods
       def authorize_host_by_client_cert(actions, _options = {})
         skip_before_action :require_login, :only => actions, :raise => false
+        skip_before_action :check_user_enabled, :only => actions
```

Comment:
hmm, it is actually not needed, because this is not defined for API controllers and this method is only for one API controller. So I belive this is good to close.

But fix would probably be:
```suggestion
        skip_before_action :check_user_enabled, :only => actions, :raise => false
```